CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Dradis Framework 3.10 (August, 2018) ##
 
 *   Add CVSS fields for Issues.
+*   Add impact, actionsToTake, requiredSkillsForExploitation, externalReferences, remedyReferences, and extrainformation fields
```

Comment:
I don't think this made it into 3.10?

Also 3.10 wasn't released in Aug unfortunately.

We probably need to update the 3.10 release date and notes, and add a 3.11 section.

Reply:
Updated!